en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -265,8 +265,6 @@ The Silverstripe framework comes with the following built-in validators:
 - [`CompositeValidator`](api:SilverStripe\Forms\Validation\CompositeValidator)
   A container for additional validators. You can implement discrete validation logic in multiple `Validator` subclasses and apply them *all* to a
   given form by putting them inside a `CompositeValidator`. The `CompositeValidator` doesn't have perform any validation by itself.
-- [`FieldsValidator`](api:SilverStripe\Forms\FieldsValidator)
-  Simply calls [`validate()`](api:SilverStripe\Forms\FormField::validate()) on all data fields in the form, to ensure fields have valid values.
```

Comment:
Does this page mention that the form does that? If not, this seems like the right page to mention it.

Reply:
Have added a mention about this at the top of the file